symphony/app/fbcnms-projects/workflows/src/proxy/proxy.js

```diff
@@ -13,77 +13,92 @@ import httpProxy from 'http-proxy';
 import logging from '@fbcnms/logging';
 import transformerRegistry from './transformer-registry';
 import {getTenantId} from './utils.js';
+import type {
+  ExpressRouter,
+  ProxyCallback,
+  ProxyNext,
+  ProxyRequest,
+  ProxyResponse,
+  TransformerEntry,
+} from '../types';
 
 const logger = logging.getLogger(module);
 const router = Router();
 router.use(bodyParser.urlencoded({extended: false}));
 router.use('/', bodyParser.json());
 
-export default async function(proxyTarget) {
-  const transformers = await transformerRegistry({proxyTarget});
+export default async function(proxyTarget: string) {
+  const transformers: Array<TransformerEntry> = await transformerRegistry({
+    proxyTarget,
+  });
 
   // Configure http-proxy
   const proxy = httpProxy.createProxyServer({
     target: proxyTarget,
     // TODO set timeouts
   });
 
-  for (const entry of transformers) {
+  for (const entry: TransformerEntry of transformers) {
```

Comment:
same here, flow can infer the type of `entry`. The flow plugins in vscode help spot these.

Reply:
done